### PR TITLE
MHP-3343: Split admin pages

### PR DESCRIPTION
**Pull request by ckozus**
https://jira.cru.org/browse/MHP-3343

### CODE REVIEW

File: app/admin/v4/translator.rb [hunk 1]
```diff
@@ -1,21 +0,0 @@
-# frozen_string_literal: true
-
-ActiveAdmin.register V4::Translator, as: 'Translators' do
```
Comment (soberstadt) [inline app/admin/v4/translator.rb:3]: Could you also remove the translator logic from the `AdminAuthorization`?

File: app/admin/v4/test_push_notifications.rb [hunk 2]
```diff
@@ -1,48 +0,0 @@
...
-  RESOURCE_NAME = 'admin_test_push_notification'
-
-  menu parent: 'Push Notifications', label: 'Send a Test Push Notification'
-
-  content do
-    render Admin::TestPushNotification.new(new_params), layout: 'active_admin'
-  end
-
-  page_action :create, method: :post do
-    @test_push_notification = Admin::TestPushNotification.new(create_params)
```
Comment (soberstadt) [inline app/admin/v4/test_push_notifications.rb:13]: Could you also remove the `TestPushNotification` model code?

Comment (ckozus) [reply app/admin/v4/test_push_notifications.rb:13]: Done!

File: app/admin/v4/translator.rb (hunk 1 above)
Comment (ckozus) [reply app/admin/v4/translator.rb:3]: Done!

File: app/admin/authorization.rb [hunk 3]
```diff
@@ -29,17 +27,6 @@ def super_admin_authorization(action, subject)
...
-    when ActiveAdmin::Page
-      true if action == :read && subject.name == 'Dashboard' && subject.namespace.name == :admin
-    when normalized(V4::LocalizedPathwayStage), normalized(V4::PushNotification::MessageTemplate), normalized(V4::ChallengeSuggestion)
-      true
-    else
-      false
-    end
-  end
-
   def authorize_push_notification_campaign?(action, subject)
```
Comment (soberstadt) [inline app/admin/authorization.rb:30]: This looks like code you should be able to remove too.

Comment (ckozus) [reply app/admin/authorization.rb:30]: @soberstadt actually I removed `authorize_push_notification_campaign?` on #3, there is a bit of overlap, once these two PRs are merged `AdminAuthorization` should be pretty cleaned up.

Comment (soberstadt) [reply app/admin/authorization.rb:30]: Ok

**Pull request merged**